<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeQueryWidget</class>
 <widget class="QWidget" name="NodeQueryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>1113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="queryTe_">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>555</y>
     <width>256</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="queryListPanel_" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>-1</y>
     <width>272</width>
     <height>739</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="queryList_"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>320</y>
     <width>108</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="exactMatchCh_">
      <property name="text">
       <string>Exact match</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="caseCh_">
      <property name="text">
       <string>Case sensitive</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="wildcardCh_">
      <property name="text">
       <string>Wildcards</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="regexpCh_">
      <property name="text">
       <string>Regexp</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTreeView" name="resTree_">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>675</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tab_">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>320</y>
     <width>329</width>
     <height>205</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="title">
     <string>Nodes</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Node type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ComboMulti" name="flagCb_"/>
     </item>
     <item row="2" column="1">
      <widget class="ComboMulti" name="typeCb_"/>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="typeResetTb_">
       <property name="toolTip">
        <string>Reset</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/reset_to_default.svg</normaloff>:/viewer/reset_to_default.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Node status:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ComboMulti" name="stateCb_"/>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="stateResetTb_">
       <property name="toolTip">
        <string>Reset</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/reset_to_default.svg</normaloff>:/viewer/reset_to_default.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Node flags:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QToolButton" name="flagResetTb_">
       <property name="toolTip">
        <string>Reset</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/reset_to_default.svg</normaloff>:/viewer/reset_to_default.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
    <zorder>label_3</zorder>
    <zorder>typeCb_</zorder>
    <zorder>typeResetTb_</zorder>
    <zorder>label_2</zorder>
    <zorder>stateCb_</zorder>
    <zorder>stateResetTb_</zorder>
    <zorder>label_4</zorder>
    <zorder>flagCb_</zorder>
    <zorder>flagResetTb_</zorder>
    <zorder>verticalSpacer_3</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="title">
     <string>Attributes</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="AttributeSearchPanel" name="attrW_" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="title">
     <string>Options</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="numCh_">
         <property name="text">
          <string>Maximum number of items:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="numSpin_"/>
       </item>
       <item>
        <widget class="QToolButton" name="serverResetTb_1">
         <property name="toolTip">
          <string>Reset</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="viewer.qrc">
           <normaloff>:/viewer/reset_to_default.svg</normaloff>:/viewer/reset_to_default.svg</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>12</y>
     <width>331</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0,0,0" columnminimumwidth="0,0,0,100,0,0">
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Node name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Search in servers:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="rootLabel_">
      <property name="text">
       <string>Search root node:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QToolButton" name="serverResetTb_">
      <property name="toolTip">
       <string>Reset</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="viewer.qrc">
        <normaloff>:/viewer/reset_to_default.svg</normaloff>:/viewer/reset_to_default.svg</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="5">
     <widget class="QLineEdit" name="rootLe_"/>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="ComboMulti" name="serverCb_"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Node path:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="5">
     <widget class="QLineEdit" name="nameLe_"/>
    </item>
    <item row="4" column="1" colspan="5">
     <widget class="QLineEdit" name="pathLe_"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>895</y>
     <width>149</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="progressLabel_">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="queryProgress_">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>865</x>
     <y>40</y>
     <width>107</width>
     <height>207</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="findPb_">
      <property name="text">
       <string>Search</string>
      </property>
      <property name="icon">
       <iconset resource="viewer.qrc">
        <normaloff>:/viewer/search_decor.svg</normaloff>:/viewer/search_decor.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopPb_">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Advanced mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="saveAsTb_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Save query as ...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="queryListTb_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Show/hide query list</string>
      </property>
      <property name="text">
       <string>Query panel</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>170</y>
     <width>111</width>
     <height>136</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>140</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Node type</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>101</width>
     <height>136</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>140</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Node Status</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>170</y>
     <width>116</width>
     <height>136</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>150</y>
     <width>52</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Node flags</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboMulti</class>
   <extends>QComboBox</extends>
   <header location="global">ComboMulti.hpp</header>
  </customwidget>
  <customwidget>
   <class>AttributeSearchPanel</class>
   <extends>QWidget</extends>
   <header>AttributeSearchPanel.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="viewer.qrc"/>
 </resources>
 <connections/>
</ui>
